tests/test_dashboard.py

```diff
@@ -12,31 +13,56 @@
 from atmo.views import server_error
 
 
-@pytest.fixture
-def dashboard_spark_jobs(now, test_user):
-    for x in range(10):
-        identifier = 'test-spark-job-%s' % x
+LAST_SPARK_JOB_ID = 0
+
+
+def create_spark_jobs(num, date, owner):
+    global LAST_SPARK_JOB_ID
```

Comment:
Hm, not a fan of this global, since we're not checking for the identifier to have this id (useless code from before) let's just use a random value here instead and get rid of the global variable.